.ci/CI/src/modules/Bootloader/Arguments.jl

```diff
@@ -278,6 +308,29 @@ function get_unit_test_julia_versions()::Vector{String}
         return ["1.10", "1.11", "rc", "nightly"]
     end
 end
+"""
+    get_integration_test_julia_versions()::Vector{String}
+
+Returns the test versions for the integration tests. If the environment variable
+CI_INTEG_TEST_VERSIONS is not set, standard versions are returned. The value of the environment
+variable is a string with the versions separated by commas. The versions are not tested for
+plausibility.
+
+CI_INTEG_TEST_VERSIONS="1.10"
```

Comment:
This is also a nice feature, it should also be documented in the docs, not just the function docs. Maybe we could have a section about environment variables that can be set, to have this customizability documented in one place?

Reply:
I wrote it in the `README.md`.